2-mandatory/2-retrying-random-numbers.js

```diff
@@ -9,8 +9,16 @@ function generateRandomNumber() {
     return Math.round(Math.random() * 100);
 }
 
-function getRandomNumberGreaterThan50() {
-    // TODO - implement using a do-while loop
+function getRandomNumberGreaterThan50()
+{
+    let total;
+    do
+    {
+        (total = generateRandomNumber());
+    }
+    while (total <= 50);
+
+    return total;
```

Comment:
I think the use of 'total' as the name of the variable creates a little confusion as we don't really calculate any total in this function